package.json

```diff
@@ -12,8 +12,9 @@
   "scripts": {
     "test": "bun test",
     "cli": "bun cli/main.ts",
-    "build": "bun build:node && bun build:web",
-    "build:node": "tsup lib/index.ts cli/main.ts --platform node --format esm --dts --sourcemap",
+    "build": "bun build:lib && bun build:cli && bun build:web",
+    "build:lib": "tsup lib/index.ts --platform node --format esm --dts --sourcemap",
+    "build:cli": "tsup cli/main.ts --platform node --format cjs --dts --sourcemap",
```

Comment:
why cjs? We should use esm for everything no?

Reply:
`commander` is internally using dynamic imports for `events` and other libs like `tty`. So, was getting error